mails/themes/classic/components/footer.html.twig

```diff
@@ -0,0 +1,8 @@
+<tr>
+  <td class="space_footer">&nbsp;</td>
+</tr>
+<tr>
+  <td class="footer" style="border-top: 4px solid #333333">
+    <span>{{ '<a href="{shop_url}">{shop_name}</a> powered by <a href="http://www.prestashop.com/">PrestaShop™</a>'|trans({}, 'EmailsBody', locale)|raw }}</span>
```

Comment:
prefer https://

Reply:
I can't in this case, this string comes from the translation catalog so I can't modify it.
But I think it would be better if this url was not hard coded in the string, unless the translations take the language into account and modify the urls, like `http://www.prestashop.com/fr`, `http://www.prestashop.com/es`, ...
@LouiseBonnard do you have any idea?

Reply:
We had a similar discussion recently with @TristanLDD because he wanted to know if you could use a tracked url at this place (which would be easier to manage if the url was a variable).
Example: `<a href="{shop_url}">{shop_name}</a> powered by <a href="{prestashop_url}">PrestaShop™</a>`

Reply:
Hum, I guess it is possible to write it as a variable. It would even be better. Is it okay from a dev point of view?

Reply:
Yes it's ok I just wanted to validate with you. However I don't think we are going to change this in this PR.
Because it requires to update some other code which manage this mail, so that the new variable is correctly injected. I think we will introduce this change in a later issue.